packages/rest/src/writer.ts

```diff
@@ -30,6 +31,16 @@ export function writeResultToResponse(
     return;
   }
 
+  // result is of type RedirectRoute
+  const isRedirect = result instanceof RedirectRoute;
```

Comment:
`instanceof` is not always reliable as the `RedirectRoute` can be loaded from different places. Let's use a TypeScript type guard function instead.

Reply:
agreed